src/improvements/tasks/MultisigTask.sol

```diff
@@ -311,6 +313,19 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager {
         vm.label(address(this), "MultisigTask");
     }
 
+    /// @notice Get the safe address string from the config file.
+    /// If the string is not found, use the value from the template.
+    function getOrReadSafeAddressString(string memory taskConfigFilePath) internal view returns (string memory) {
```

Comment:
I could have just override `safeAddressString()` with this logic, but that felt a bit sneaky to do.